src/scss/layout/_anchor-link.scss

```diff
@@ -0,0 +1,59 @@
+@use '~accoutrement/sass/tools';
+
+.anchor-link-wrapper {
+  display: inline-block;
+  position: relative;
+
+  h2 {
+    display: inline-block;
+    padding-left: var(--anchor-link-padding-l, var(--gutter));
+
+    @include tools.above('wide-page') {
+      --anchor-link-padding-l: 0;
+    }
+  }
+}
+
+.header-anchor {
+  position: absolute;
+  left: var(--anchor-position-offset-l, -0.25em);
+  top: var(--anchor-position-offset-t, var(--gutter-plus));
+  transform-origin: top center;
+  width: var(--gutter);
+
+  @include tools.above('wide-page') {
+    --anchor-position-offset-l: -1em;
+    --anchor-position-offset-t: 1.25em;
+  }
+
+  &.anchor-clicked {
+    @include tools.transition('move');
+
+    svg {
+      transform: rotate(-45deg);
+    }
+  }
+
+  &:visited {
+    --fill: var(--active);
+  }
+}
+
+.anchor-clicked::before {
+  background: var(--feature);
+  border-radius: var(--radius);
+  color: var(--bg);
+  content: 'Link Copied';
+  font-weight: bold;
+  font-size: var(--xxsmall);
+  left: var(--anchor-text-offset-l, -0.25em);
+  padding: var(--quarter-shim) var(--half-shim);
+  position: absolute;
+  top: var(--anchor-text-offset-t, -2.25em);
+  white-space: nowrap;
+
+  @include tools.above('wide-break') {
+    --anchor-text-offset-l: -6.5em;
+    --anchor-text-offset-t: 0.5em;
+  }
```

Comment:
@dvdherron there are a number of numbers 😵‍💫 used in this file. Are any related? I see two that are -.25em ... would it make sense to make a few variables?

Reply:
@stacyk  I tried to clean this up by making some variables and simplifying what happens on widescreens. I think it works fine if "Link Copied" stays on top of the icon no matter the screen width?